Model.cpp

```diff
@@ -92,41 +100,46 @@ int Model::insertBatch(vector<Model*> models, int batchsize, Mode mode)
             if (!ignoreField) {
                 paramNum++;
                 
-                if (f->getType() == INTEGER)
-                {
-                    IntegerField * i = static_cast <IntegerField*>(f);
-                    if (i->isNull()) {
-                        conn.setNull(paramNum);
-                    } else {
-                        conn.setInt(paramNum, i->getValue());
-                    }
-                }
-                else if(f->getType() == FLOAT)
-                {
+                if (f->isNull()) {
+                    conn.setNull(paramNum);
+                } else {
```

Comment:
You can remove the `else` block if you use a `continue` statement in the `if` block after setting the parameter to null.